Solutions/Tomcat/Analytic Rules/TomcatSQLiPattern.yaml

```diff
@@ -0,0 +1,28 @@
+id: ce84741e-4875-11ec-81d3-0242ac130003
+name: Tomcat - Sql injection patterns
+description: |
+  'Detects possible sql injection patterns'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+query: |
+  TomcatEvent
+  | where UrlOriginal contains "1/*'*/" or UrlOriginal contains "1'||'asd'||'" or UrlOriginal contains "'1'='1" or UrlOriginal contains "1' or '1'='1" or UrlOriginal contains "1 or 1=1" or UrlOriginal contains "1=1" or UrlOriginal contains "1/*!1111'*/" or UrlOriginal contains "'or''='"

```

Comment:
Same here on using CONTAINS and OR

Reply:
It is not reasonable to use `has/has_any` here